openshift-kube-apiserver/admission/customresourcevalidation/route/validation_wrapper_test.go

```diff
@@ -0,0 +1,41 @@
+package route
+
+import (
+	"fmt"
+	"testing"
+
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+// setter is only available in unit-tests
+func (a *validateCustomResourceWithClient) setupWithFakeClient() {
+	c := fake.NewSimpleClientset()
+	a.secretsGetter = c.CoreV1()
+	a.sarGetter = c.AuthorizationV1()
+	a.routeValidationOptsGetter = NewRouteValidationOpts()
+}
+
+// TestRouteValidateInitialization will check if validator client is
+// conformant with admission.InitializationValidator interface
+func TestRouteValidateInitialization(t *testing.T) {
+	admitter, err := NewValidateRoute()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	routeAdmitter, ok := admitter.(*validateCustomResourceWithClient)
+	if !ok {
+		t.Fatal(fmt.Errorf(
+			"could not type cast returned value of NewValidateRoute() into type validateCustomResourceWithClient, " +
+				"perhaps you changed the type in the implementation but not in the tests!"))
```

Comment:
No need for `fmt.Errorf` here:
```suggestion
		t.Fatal("could not type cast returned value of NewValidateRoute() into type validateCustomResourceWithClient, " +
			"perhaps you changed the type in the implementation but not in the tests!")
```

Reply:
updated it.